modules/dropdown/js/dropdown_directive.js

```diff
@@ -158,9 +157,12 @@
             }
             enableBodyScroll = undefined;
 
+            var dropdownToggleElement;
             if (lxDropdown.hasToggle)
             {
-                dropdownToggle
+                dropdownToggleElement = (angular.isString(dropdownToggle)) ? angular.element(dropdownToggle) : dropdownToggle;
```

Comment:
Can we not compute this only once in the registerDropdownToggle method? So we don't have to re-wrap it with angular.element every time we close the dropdown or get the available height?